PyInstaller/utils/conftest.py

```diff
@@ -541,7 +535,10 @@ def pyi_builder(tmpdir, monkeypatch, request, pyi_modgraph):
 @pytest.fixture
 def pyi_builder_spec(tmpdir, request, monkeypatch, pyi_modgraph):
     # Save/restore environment variable PATH.
-    monkeypatch.setenv('PATH', os.environ['PATH'], )
+    monkeypatch.setenv(
+        'PATH',
+        os.environ['PATH'],
+    )
```

Comment:
Would have `yapf` decided differently here had original line not contain redundant trailing comma?